content/en/database_monitoring/query_metrics.md

```diff
@@ -105,7 +105,7 @@ Click the **Metrics** tab to see more graphs of metrics for this query.
 
 Datadog collects explain plans continuously, so a given query can have multiple plans. Those plans are normalized and shown separately so that you can see if a query has plans that perform better or have higher relative cost than others. For example, the following shows two explain plans for a query, and one has much lower average latency:
```

Comment:
(Trying to reword the sentence above from `two plans` to `one plan`)

```suggestion
Datadog collects explain plans continuously, so a given query can have multiple plans. Those plans are normalized and shown separately so that you can see if a query has plans that perform better or have higher relative cost than others. For example, the following explain plan contains information for a query:
```